docs/users.rst

```diff
@@ -629,6 +629,27 @@ flags
   List of flags to pass to the cachito request. See the cachito_ documentation
   for further reference.
 
+Once the `remote_source` map described above is set in ``container.yaml``, you
+can now extract the upstream sources (with bundled dependencies) archive
+provided by cachito in your build image by adding::
+
+    RUN mkdir -p $REMOTE_SOURCE_DIR
+    ADD $REMOTE_SOURCE $REMOTE_SOURCE_DIR
+
+to your Dockerfile.
```

Comment:
I'm curious if OSBS could do this Dockerfile change for user

Reply:
The tricky part is that this is a multistage build. If we do so, should we add them in the first stage? Should we remove the sources dir in the final stage as well?

Reply:
I see.
In multistage builds we don't have to remove anything, builds will be discarded unless user copy results to final build